.github/workflows/integration-package-tests.yaml

```diff
@@ -3,7 +3,9 @@ name: Integrations Packages Tests
 on:
   pull_request:
     paths:
-      - "src/integrations/*/**"
+      - .github/workflows/integration-package-tests.yaml
+      - "src/**/*"
```

Comment:
was this meant to be only on `.py` files? I'm surprised that my changes to the `prefect-ray` readme are running this workflow
https://github.com/PrefectHQ/prefect/pull/13630/files

Reply:
for sure!